argo/template/dm.yaml

```diff
@@ -0,0 +1,76 @@
+metadata:
+  name: tipocket-dm
+  namespace: argo
+spec:
+  templates:
+    - name: tipocket-dm
+      inputs:
+        parameters:
+          - name: ns
+            default: tipocket-dm
+          - name: hub
+            default: "docker.io"
+          - name: repository
+            default: pingcap
+          - name: image_version
```

Comment:
In the previous PR, we almost change all parameters like image_version to image-version, including storage_class, run_time and etc.

Reply:
renamed in https://github.com/pingcap/tipocket/pull/220/commits/44941bc7c203f226fba4676a32197d038630c4b8.